### DISCUSSION TITLE
Dynamic routing under the problem of obtaining parameters

### DISCUSSION THREAD

**Discussion by zjcwill** [ElMassimo/iles#19] [Q&A] [answered]
How to get path and Params correctly in the page.

```vue
   <router-link
          :to="{
            path: '/static_article',
            params: { cate: 'book' },
            query: { id: 3 },
          }"
          
          >query</router-link
        >
```
The above writing does not take effect.

Do we have something like this？https://nextjstips.com/tip/ssg-with-dynamic-routes

**Comment by ElMassimo** [comment, marked as answer]
__Edit:__ @zjcwill `iles@0.4.2` added support for dynamic parameters + [`getStaticPaths`](https://iles-docs.netlify.app/guide/routing#getstaticpaths).

---

__Old Answer:__

Hi! At the moment îles does not support dynamic paths.

Planning to add a similar API soon, which might become available next week.

In the meantime, I would [recommend using Astro](https://iles-docs.netlify.app/faqs#astro) instead.
